bowling/tests/bowling_tests_gt.cpp

```diff
@@ -0,0 +1,200 @@
+    std::vector<Frame> expectedPlayerRolls{{'1', '2'},
+                                           {'3', '4'},
+                                           {'5', '6'},
+                                           {'7', '8'},
+                                           {'9', '9'},
+                                           {'X', ' '},
+                                           {'0', '-'},
+                                           {'5', '/'},
+                                           {'5', '4'},
+                                           {'X', ' '}};
+    file << "Name:12|34|56|78|99|X|0-|5/|54|X||\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getPlayers()[0].rolls, expectedPlayerRolls);
+}
+
+std::ostream& operator<<(std::ostream& os, Game::Status status) {
+    os << static_cast<int>(status);
+    return os;
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnNoGameWhenFileIsEmpty) {
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::NO_GAME);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnInProgressWhenFramesAreNotFull) {
+    file << "Robcio:12|27|X\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::IN_PROGRESS);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnFinishedWhenFramesAreFull) {
+    file << "Robcio:12|27|X|1/|22|11|0-|0-|12|0-||\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::FINISHED);
+}
+
+TEST_F(GameTests, countPointsFromVectorOfFramesWithoutStrikeNorSpareIncompleteGame) {
+    std::vector<Frame> playerRolls{{'1', '2'}, {'4', '5'}, {'2', '2'}, {'7', '0'}};
+    EXPECT_EQ(game.countPoints(playerRolls), 23);
+}
+
+TEST_F(GameTests, countPointsFromVectorOfFramesWithStrikeWithoutSpareIncompleteGame) {
+    std::vector<Frame> playerRolls{{'X', ' '}, {'4', '5'}, {'X', ' '}, {'X', ' '}, {'1', '1'}, {'X', ' '}};
+    EXPECT_EQ(game.countPoints(playerRolls), 72);
+}
+
+TEST_F(GameTests, countPointsFromVectorOfFramesWithSpareWithoutStrikeIncompleteGame) {
+    std::vector<Frame> playerRolls{{'2', '/'}, {'4', '5'}, {'3', '/'}, {'2', '/'}, {'0', '/'}};
+    EXPECT_EQ(game.countPoints(playerRolls), 55);
+}
+
+TEST_F(GameTests, countPointsFromVectorOfFramesCompleteGameWithStrikeSpareAndExtraFrame) {
+    std::vector<Frame> playerRolls{{'X', ' '}, {'4', '5'}, {'X', ' '}, {'X', ' '}, {'1', '1'},
+                                   {'X', ' '}, {'5', '/'}, {'X', ' '}, {'1', '8'}, {'X', ' '}, {'5', '1'}};
+    EXPECT_EQ(game.countPoints(playerRolls), 152);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnFinishedWhenExtraFrames) {
+    file << "Robcio:12|27|X|1/|22|11|0-|0-|12|X||17\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::FINISHED);
+}
+
+TEST_F(GameTests, GameOutputCanBePrintedOnScreen) {
+    game.printOutput(std::cout, 1);
+}
```

Comment:
not so useful test. It doesn't test anything. Any implementation provided as printOutput would pass.